source/clear-linux/guides/maintenance/swupdaddpkg.rst

```diff
@@ -1,150 +1,156 @@
 .. _swupdaddpkg:
 
-Create and add custom bundles to your upstream Clear Linux System
+Create and add custom bundles to your upstream Clear Linux system
 #################################################################
 
 |CLOSIA| offers many curated bundles that you can install on your system to
 create your desired capabilities. If the available upstream bundles do not
-meet your needs, you can create and add your own custom bundles to your
-system using one of two methods. Note: Upstream refers to the official version of |CL|.
+meet your needs, you can create and add your own custom bundles to your system
+using one of two methods. Note: Upstream refers to the official version of
+|CL|.
 
 The first method is to use the :ref:`mixer tool<mixer>` to create your own
-|CL| image and adding your bundles to it.  Mixing your own |CL| image can
-give you great control and flexibility, but you have to become your own OSV
-and maintain your own releases and updates because you’ve forked from
-upstream.  
-
-The second method is to use the :command:`swupd-add-pkg` tool, which also makes
-use of mixer to create custom bundles that you can add to your upstream |CL|
-system.  In comparison, this method is simpler. It is a “light” forking from
-upstream, which means you can continue to get upstream bundles and updates.
-Also, if needed, you can revert your system back to 100% upstream in a
-pinch. 
-
-This guide shows you how to accomplish the second method by following these
+|CL| image and add your bundles to it. Mixing your own |CL| image can
+give you great control and flexibility, however, you must act as an OSV
+and maintain your releases and updates because you have forked from
+upstream.
+
+The second method is to use the :command:`swupd-add-pkg` tool, which also uses
+mixer to create custom bundles that you can add to your upstream |CL| system.
+This simpler method provides a “light” forking from upstream, which means you
+can continue to get upstream bundles and updates. If needed, you can revert
+your system back to the upstream version easily.
+
+This guide describes how to accomplish the second method by following these
 steps:
 
-#. Set up the workspace
-#. Copy your custom RPM package to the workspace
-#. Create a bundle with your custom RPM package
-#. Migrate your |CL| system to your custom mix
-#. Add your custom bundle to your system
-#. Optional: Revert your system back to 100% upstream
+#. Set up the workspace.
+#. Copy your custom RPM package to the workspace.
+#. Create a bundle with your custom RPM package.
+#. Migrate your |CL| system to your custom mix.
+#. Add your custom bundle to your system.
+#. Optional: Revert your system back to 100% upstream.
 
 Set up the workspace
 ********************
 
-#. Log in and get root privileges: 
+#. Log in and get root privileges.
```

Comment:
No logging into root :-1: :-1:  Use sudo where it's needed:
`sudo swupd-add-pkg foo bar`